test/python/WMCore_t/JobStateMachine_t/ChangeState_t.py

```diff
@@ -4,6 +4,9 @@
 
 """
 
+from builtins import range, int, str as newstr
+from future.utils import viewitems, viewvalues, listvalues
```

Comment:
pylint reports listvalues and viewitems as unused imports here. Shall we remove them?